src/ProcessManagerBundle/Factory/ProcessFactory.php

```diff
@@ -45,8 +45,14 @@ public function createProcess(
         string $type = null,
         string $message = '',
         int $total = 1,
-        int $progress = 0
+        int $progress = 0,
+        int $started = -1,
+        int $completed = 0
     ) {
-        return new $this->model($name, $type, $message, $total, $progress);
+        if($started == -1){
+            $started = new \DateTime();
+            $started = $started->format('U');
```

Comment:
would just time() be fine?

Reply:
Interface: good catch.

trailing comma: So that the diff looks cleaner once somebody else changes something. but forget about that, thats PHP7.3 and not 7.1

Reply:
Oh... wrong discussion -> consider this for the discussion above pls.